doc/source/ray-overview/examples/web-crawler.ipynb

```diff
@@ -0,0 +1,241 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Speed up your web crawler by parallelizing it with Ray"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "In this example we'll quickly demonstrate how to build a simple web scraper in Python and\n",
+    "parallelize it with Ray Tasks with minimal code changes.\n",
+    "\n",
+    "To run this example locally on your machine, please first install `ray` and `beautifulsoup` with\n",
```

Comment:
let's also specify the version you tested against?

Reply:
done